base/applications/mscutils/servman/export.c

```diff
@@ -34,7 +34,8 @@ GetTextFromListView(PMAIN_WND_INFO Info,
 
 static BOOL
 SaveServicesToFile(PMAIN_WND_INFO Info,
-                   LPCWSTR pszFileName)
+                   LPCWSTR pszFileName,
+                   DWORD nIndex)
```

Comment:
Rename this to something like "nFormat" or something, in order for code readers to know what this "index" is about.
Besides, it needs to be validated and be between 0 and 1 (or 1 and 2, depending on how you handle it later), for the `seps[nIndex-1]` done later to not crash.